src/part-guide/concurrency.md

```diff
@@ -1,12 +1,12 @@
+The goal of this chapter is to give you a high-level idea of how async concurrency works and how it is different from concurrency with threads. Forming a good mental model of what is going on will help before getting in to the practicalities. If you're the kind of person who likes to see some real code first, you might like to read the next chapter or two and then come back to this one.
 
 We'll start with some motivation, then cover [sequential programming](#sequential-execution), [programming with threads or processes](#processes-and-threads), and then [async programming](#async-programming). The chapter finishes with a section on [concurrency and parallelism](#concurrency-and-parallelism).
 
 Users want their computers to do multiple things. Sometimes users want to do those things at the same time (e.g., be listening to a music app at the same time as typing in their editor). Sometimes doing multiple tasks at the same time is more efficient (e.g., getting some work done in the editor while a large file downloads). Sometimes there are multiple users wanting to use a single computer at the same time (e.g., multiple clients connected to a server).
 
-To give a lower-level example, a music program might need to keep playing music while the user interacts with the user interface (UI). To 'keep playing music', it might need to stream music data from the server, process that data from one format to another, and send the processed data to the computer's audio system via the operating system (OS). For the user, it might need to send and receive data or commands to the server in response to the user instructions, it might need to send signals to the subsystem playing music (e.g., if the user changes track or pauses), it might need to update the graphical display (e.g., highlighting a button or changing the track name), and it must keep the mouse cursor or text inputs responsive while doing all of the above.
+To give a lower-level example, a music program might need to keep playing music while the user interacts with the user interface (UI). To 'keep playing music', it might need to stream music data from the server, process that data from one format to another, and send the processed data to the computer's audio system via the operating system (OS). For the user interface, it might need to send and receive data to the server in response to the user instructions; it might need to send signals to the subsystem playing music (e.g., if the user changes track or pauses); it might need to update the graphical display (e.g., highlighting a button or changing the track name); and it must keep the mouse cursor or text inputs responsive while doing all of the above.
```

Comment:
Why remove 'commands' here?

Reply:
Because the sentence is really long and I don't think "or commands" adds much